CONTRIBUTING.md

```diff
@@ -97,17 +97,16 @@ you can enable the docker tests by setting a gradle property of
 ``"enable.docker.tests"`` to true. See the gradle.properties file in the root of the project
 for more details.
 
-2. From the root project directory, initialize repository dependencies
+2. Clone the repository recursively
 
-   `make init-git-submodules`
+    `git clone git@github.com:<username>/opentelemetry-java.git --recursive`
 
-3. Run the following commands to build, run tests and most static analysis, and
-check formatting:
-
-    ##### OS X or Linux
+or alternatively initialize submodules for an existing clone.
 
-    `make test verify-format`
+   `git submodule init`
+   `git submodule update`   
 
-    ##### Windows
+3. Run the following commands to build, run tests and most static analysis, and
+check formatting:
 
-    `gradlew.bat clean assemble check verGJF`
\ No newline at end of file
+    `./gradlew check`
```

Comment:
I didn't separate out a Windows command since I generally don't see that in READMEs, I think users are used to switching to the bat version for these sort of commands.